.github/actions/set-env-variables/action.yml

```
@@ -16,7 +16,14 @@ runs:
         echo "CILIUM_CLI_VERSION=$CILIUM_CLI_VERSION" >> $GITHUB_ENV
         echo "CILIUM_CLI_IMAGE_REPO=quay.io/cilium/cilium-cli-ci" >> $GITHUB_ENV
         echo "CILIUM_CLI_SKIP_BUILD=true" >> $GITHUB_ENV
-        echo "CILIUM_CLI_CODE_OWNERS_PATHS=CODEOWNERS" >> $GITHUB_ENV
+
+        # Use TESTOWNERS file if it exists, otherwise fall back to CODEOWNERS only
+        CILIUM_CLI_CODE_OWNERS_PATHS="CODEOWNERS"
+        if [[ -f "TESTOWNERS" ]]; then
+          CILIUM_CLI_CODE_OWNERS_PATHS="TESTOWNERS,CODEOWNERS"
```

I think it should be possible to do this? In particular, having the extra file at the end would be good.
```suggestion
          CILIUM_CLI_CODE_OWNERS_PATHS+="TESTOWNERS"
```

---

Yeah, this is better.